.vscode/settings.json

```diff
@@ -1,3 +1,5 @@
 {
-  "files.insertFinalNewline": true
+  "files.insertFinalNewline": true,
+// "spellright.language": ["en_US"],
```

Comment:
Not legal json

Reply:
Its legal and normal in vscode though :D blame microsoft :D